<launch>
  <node pkg="sick_safevisionary_driver" type="sick_safevisionary_driver_node" name="sick_safevisionary" output="screen">
    <!-- UDP port of Measurement Data Stream, default: 6060 -->
    <param name="udp_port" value="6060" />
    <param name="frame_id" value="camera" />
  </node>
</launch>
